src/styles/card.css

```diff
@@ -0,0 +1,110 @@
+/* #region card */
+:root {
+  font-family: Roboto, sans-serif;
+
+  --secondary-text-color: #616070;
+  --button-color: #00acdc;
+  --main-border-radius: 5px;
+  --text-color: #060b35;
+}
+
+body,
+h2,
+p,
+div {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
+  border: solid #f3f3f3 1px;
+  border-radius: var(--main-border-radius);
+}
+
+.card__imac-img {
```

Comment:
Consider using a more descriptive name for the image class instead of 'card__imac-img'. For example, 'card__image'.